docs/help/contents/self-hosting/getting-started.md

```diff
@@ -0,0 +1,295 @@
+# Getting started
+
+## Minimum server requirements
+
+- **Operating system**: Linux
+- **RAM**: 1 GB
+- **Storage**: 20 GB
+- **CPU**: any x64 or ARM CPU will do
+
+## Prerequisites
+
+1. Docker
+2. Docker Compose
+3. wget
+4. curl
+5. (Optional) Caddy or Nginx for reverse proxying
+
+You **must** have a working Linux server with SSH access.
+
+> warn Disclaimer
+>
+> How you secure or setup your server is up to you, and this guide will NOT provide any recommendations outside what is directly relevant to Notesnook.
+>
+> Self hosting means managing everything on your own so **we will not be responsible if you lose any data** during setup, upgrade, or any other step. It is recommended that you create regular backups in both the app and on the server to avoid this.
+
+## Installation
+
+1. Create a directory where configuration files can be placed:
+   ```sh
+   mkdir notesnook-sync-server
+   ```
+2. Navigate into the newly created directory:
+   ```sh
+   cd notesnook-sync-server
+   ```
+3. Download the `docker-compose.yml` file:
+   ```sh
+   wget https://raw.githubusercontent.com/streetwriters/notesnook-sync-server/master/docker-compose.yml
+   ```
+4. Download the `.env` file:
+   ```sh
+   wget https://raw.githubusercontent.com/streetwriters/notesnook-sync-server/master/.env
+   ```
+
+You should end up with the following files in the `notesnook-sync-server` directory:
+
+- docker-compose.yml
+- .env
+
+## Configuration
```

Comment:
Minor Nit: For convenience, I think the headings in this section should follow the same order as the variables in the env file.